openfisca_france/model/cotisations_sociales/travail_prive.py

```diff
@@ -472,9 +472,9 @@ def function(self, simulation, period):
         taux_reduit = simulation.legislation_at(period.start).forfait_social.taux_reduit
 
         # TODO: complete this
-        assiette_taux_plein = 0 # TODO: complete this
-        assiette_taux_reduit = prevoyance_obligatoire_cadre + prise_en_charge_employeur_prevoyance_complementaire
-
+        assiette_taux_plein = 0 # TODO: complete this        
+        assiette_taux_reduit = prevoyance_obligatoire_cadre - prise_en_charge_employeur_prevoyance_complementaire
+        
```

Comment:
@benjello Could you set up their environment to automatically remove trailing whitespace?  :)
If it sounds costly, we'll see together how to do that.